Automattic/jetpack
# Likes Widget: Fix accessibility on likes popover

Related to https://github.com/Automattic/wp-calypso/issues/85710

## Proposed changes:

* Fixes accessibility on likes popover
* Changes should match with changes on D133146-code

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:

* Should be tested together with D133146-code
* Apply this PR to your local
* Spin up a JT site
* A post with likes on your JT site
* Navigate to the `x likes` button using tabs
* Click to open the popover and continue navigating with tabs
* The focus should move to the likes popover
* Leave the popover with tabs
* It should continue to the next focusable element after the likes iframe
* Enable the VoiceOver and follow the same steps
* You should have a similar result, being able to access the likers inside the popover and continue navigating on the page
